tactics/cbn.ml

```diff
@@ -38,34 +39,34 @@ module Cst_stack = struct
 - constant applied to params = term in head applied to args
 - there is at most one arguments with an empty list of args, it must be the first.
 - in args, the int represents the indice of the first arg to consider *)
-  type t = (constr * constr list * (int * constr array) list)  list
+  type t = (constr * constr list * (int * constr array) list * volatile)  list
 
   let empty = []
-  let is_empty = CList.is_empty
+  let all_volatile = CList.for_all (fun (_,_,_,{volatile}) -> volatile)
```

Comment:
CI errors for hott and relation_algebra in the first run seem to have been because of this mising change.